Modules/Core/Filament/Admin/Resources/Users/UserResource.php

```
@@ -55,4 +57,29 @@ public static function getPages(): array
+
+    public static function canView(Model $record): bool
+    {
+        return auth()->user()?->can(Permission::VIEW_USERS->value) ?? false;
+    }
+
+    public static function canEdit(Model $record): bool
+    {
+        return auth()->user()?->can(Permission::EDIT_USERS->value) ?? false;
+    }
+
+    public static function canDelete(Model $record): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$record'. (undefined)

(UnusedFormalParameter)